src/Common/src/CoreLib/System/IO/Path.Windows.cs

```diff
@@ -5,11 +5,11 @@
 using System.Diagnostics;
 using System.Text;
 
-#if MS_INTERNAL_IO
+#if MS_IO_REDIST
 using System;
 using System.IO;
 
-namespace Microsoft.Internal.IO
+namespace Microsoft.IO.Redist
```

Comment:
The EventSource prior art is using `Redist` just for the package name, I think. The  namespaces do not have the Redist suffix.

Reply:
E.g. https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/Diagnostics/Tracing/TraceLogging/ConcurrentSet.cs#L8

Reply:
Ok.